charts/tidb-operator/values.yaml

```diff
@@ -50,5 +50,5 @@ scheduler:
     requests:
       cpu: 80m
       memory: 50Mi
-  # modify the hyperkube verison to your kubernetes version
-  kubeSchedulerImage: gcr.io/google-containers/hyperkube:v1.10.12
+  # this hyperkube verison must be the same as your kubernetes cluster version
+  kubeSchedulerImage: gcr.io/google-containers/hyperkube:v1.12.1
```

Comment:
Comment this out and make this field `required`, so users have to fill this out when installing tidb-operator.